doctrine/associations.rst

```diff
@@ -50,10 +87,20 @@ property on the ``Product`` class, annotated as follows:
             // ...
 
             /**
-             * @ORM\ManyToOne(targetEntity="Category", inversedBy="products")
-             * @ORM\JoinColumn(name="category_id", referencedColumnName="id")
+             * @ORM\ManyToOne(targetEntity="App\Entity\Category", inversedBy="products")
+             * @ORM\JoinColumn(nullable=true)
```

Comment:
it is`nullable=true` by default for `@ORM\JoinColumn` annotation.

Reply:
Yea... but I *do* want it to be nullable (it just makes the example easier, no error in case you already have Products in the database) but I also want to show this option so that people are thinking about it.